b288702/README.md

```diff
@@ -1,4 +1,28 @@
-# Phase 2 Validation
+# Mainnet Circuits
+
+Circuits frozen at `rust-fil-proofs` commit: `b288702`
+
+1. SDR-PoRep-32GiB (Poseidon)
+2. SDR-PoRep-64GiB (Poseidon)
+3. Winning-PoSt-32GiB (Poseidon)
+4. Winning-PoSt-64GiB (Poseidon)
+5. Window-PoSt-32GiB (Poseidon)
+6. Window-PoSt-64GiB (Poseidon)
+
+### Participation Requirements
+
+The following are requirements for participating in Filecoin's mainnet Phase 2. Note that runtime varies inversely with the number of CPU threads available.
+
+| Circuit            | RAM Req. | Disk Space Req. | Runtime 96 CPU Threads | Runtime 8 CPU Threads |
+| :------------------ | :--------: | :---------------: | :-----------------------: | :---------------------: |
+| SDR-PoRep-32GiB    | 6GB      | 52GB            | 45m                    | 6h                    |
```

Comment:
I don't think these numbers are correct. My machine ran out of RAM and I have 32GiB (I had other things running, but I needed more than 6GiB).

Also the disk space doesn't seem to be correct. The phase 1.5 parameters are alone >70GiB.

Reply:
oh wait, yeah this is wrong - I think this is the contribution req.